constants:
  FRAME_WIDTH : 5.0  # Frame size in simulation units
  FRAME_HEIGHT : 5.0
  FRAME_DISCRETIZATION : 0.1  # Step size
  SPEED : 0.2  # Weed movement speed
  ROTATION_ANGLE : 10  # Rotation in degrees
  SCREEN_WIDTH : 500  # Pixel width of the window
  SCREEN_HEIGHT : 500  # Pixel height of the window
  FONT_SIZE : 16

